### PR TITLE
chore: [CHAOS-2476]: Added release notes docs for 1.19.x Chaos module release

### PR DESCRIPTION
**Pull request by Jonsy13**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by ajeshbaby — APPROVED** [review]
LGTM

File: release-notes/chaos-engineering.md [hunk 1]
```diff
@@ -18,10 +18,55 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
 :::
 
-## Latest: Version 1.18.7
+## Latest: Version 1.19.2
 
 ### What's new
 
+* Added Asupport for Authentication & HTTPs in HTTP Probes for Kubernetes chaos faults. (CHAOS-2381)
```
Comment (ajeshbaby) [inline release-notes/chaos-engineering.md:25]: Added support for Authentication & HTTPs in HTTP Probes for Kubernetes chaos faults. (CHAOS-2381)

Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Thanks for pointing this one. Fixed now!

File: release-notes/chaos-engineering.md [hunk 2]
```diff
@@ -18,10 +18,55 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* Added support for authentication and TLS in Prometheus probes in Kubernetes chaos faults. (CHAOS-2295)
+
+* Chaos Studio no longer shows ChaosHubs with no experiments/faults during experiment creation. (CHAOS-2283)
+
+* A new option has been added to preserve or delete the chaos experiment resources with a single toggle. Experiment resources can be preserved for debugging purposes. (CHAOS-2255)
+
+* The Docker Service Kill chaos fault was enhanced to support containerd service as well. Users can select the type of service via a new tunable (SERVICE_NAME) they want to kill. (CHAOS-2220)
+
+* Added support for experiment run specific manifest download from auxiliary nav when viewing specific run details. (CHAOS-1832)
```
Comment (KimberlyFields) [inline release-notes/chaos-engineering.md:37]: @Jonsy13 this bullet item is confusing. Can you please revise for clarity?

File: release-notes/chaos-engineering.md [hunk 3]
```diff
@@ -18,10 +18,55 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
 :::
 
-## Latest: Version 1.18.7
+## Latest: Version 1.19.2
 
 ### What's new
 
+* Added support for Authentication and HTTPs in HTTP Probes for Kubernetes chaos faults. (CHAOS-2381)
```
Comment (neelanjan00) [inline release-notes/chaos-engineering.md:25]:
```suggestion
* Added support for Authentication and SSL certifcate in HTTP Probes for Kubernetes chaos faults. (CHAOS-2381)
```

File: release-notes/chaos-engineering.md [hunk 4]
```diff
@@ -18,10 +18,55 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+### Early access
+
+* Linux Chaos Faults (This feature is currently behind a feature flag named `CHAOS_LINUX_ENABLED`)
+    - Added support for targeting multiple network interfaces in network faults. (CHAOS-2349)
+    - The script generated to add the Linux infrastructure had incorrect flags due to changes in terminologies. This has now been corrected to reflect updated installation flags. (CHAOS-2313)
+
+* Resilience Probes (This feature is currently behind a feature flag named `CHAOS_PROBES_ENABLED`)
+    - Users had to select the **Setup Probe** button 2 times. It should now work only with a single click. It was dependent on formik validations, which in turn was halting the functionality of handleSubmit due to incorrect Yup validations. (CHAOS-2364)
+   - When using the same probes in two faults under same chaos experiment, Probe API was returning the probe two times in the second fault. This was due to probeNames being a global variable and using the same probe name multiple times was causing the name to be appended without re-initializing the variable. Scoping it down to local scope fixed this issue. (CHAOS-2452)
```
Comment (neelanjan00) [inline release-notes/chaos-engineering.md:47]:
```suggestion
   - When using the same probes in two faults under same chaos experiment, Probe API was returning the probe two times in the second fault. This was due to `probeNames` being a global variable and using the same probe name multiple times was causing the name to be appended without re-initializing the variable. Scoping it down to local scope fixed this issue. (CHAOS-2452)
```

**Review by neelanjan00 — APPROVED** [review]

File: release-notes/chaos-engineering.md (hunk 2 above)
Comment (Jonsy13) [reply release-notes/chaos-engineering.md]: Fixed now, Please check!

**Review by neelanjan00 — DISMISSED** [review]

**Review by KimberlyFields — APPROVED** [review]

**Pull request merged**